### PR TITLE
(feat): follow lowercase convention for buffer properties

### PR DESCRIPTION
**Pull request by d12frosted**
##### Motivation for this change

See https://github.com/org-roam/org-roam/pull/769 for more information

### CODE REVIEW

**Comment by d12frosted** [comment]
Oh my, I've broken tests.

**Comment by d12frosted** [comment]
Running tests locally on macos is painful. Still looking which non GNU bin is the cultprit. 

```
λ ./makem.sh -vv --sandbox $SANDBOX_DIR test
\e[0;34mLOG (2020-11-20 09:16:36): Running all tests...\e[0m
./makem.sh: line 553: ${test_name,,}-tests-p: bad substitution
\e[0;32mLOG (2020-11-20 09:16:36): Finished with 0 errors and 0 non-blocking errors.\e[0m
```

**Comment by d12frosted** [comment]
Ah found it, now something different:

```
$ ./makem.sh -vv --sandbox $SANDBOX_DIR test
LOG (2020-11-20 09:23:29): Running all tests...
LOG (2020-11-20 09:23:29): ERT tests not found.
LOG (2020-11-20 09:23:29): Compiling...
LOG (2020-11-20 09:23:30): Compiling finished without errors.
LOG (2020-11-20 09:23:30): Running Buttercup tests...
Running 27 out of 28 specs.

Cache Build
  cache build from scratch time to be acceptableContacting host: github.com:443
(Shell command succeeded with no output)
(org-roam) Processed 0/1000 modified files...
Lisp nesting exceeds ‘max-lisp-eval-depth’
ERROR (2020-11-20 09:23:38): Buttercup tests failed.
LOG (2020-11-20 09:23:38): Finished with 1 errors and 0 non-blocking errors.
```

**Comment by d12frosted** [comment]
Ah, it's the same as on CI

File: org-roam-doctor.el [hunk 1]
```diff
@@ -102,7 +102,7 @@ AST is the org-element parse tree."
     (org-element-map ast 'keyword
       (lambda (kw)
         (let ((key (org-element-property :key kw)))
```
Comment (jethrokuan) [inline org-roam-doctor.el:104]: `org-element` return keys that are always uppercased, so this is incorrect, I think.

**Review by jethrokuan — COMMENTED** [review]
You've hit some of the weird snags of `org-element`, the same mistake I made back when I downcased things. I think the way to go about this is to simply `org-roam--set-global-prop` instead.

**Comment by d12frosted** [comment]
I need more time :) sorry for opening PR and then doing nothing here. I am yet to understand what exactly leads to `exceeds ‘max-lisp-eval-depth’`.

**Comment by jethrokuan** [comment]
Superceded by #1352

**Pull request closed without merging**